src/hotspot/share/opto/phasetype.hpp

```diff
@@ -157,11 +156,19 @@ class PhaseNameIter {
 
 class PhaseNameValidator {
  private:
+  CHeapBitMap _mask;
   bool _valid;
+  bool _set;
```

Comment:
I suggest to also name this field `_mask_set` to match the accessor function `is_mask_set()`. 
```suggestion
  bool _mask_set;
```